ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,464 +1,476 @@
+{%         endif %}
+{%         for network in router_bgp.vrfs[vrf].networks | arista.avd.natural_sort %}
+{%             if router_bgp.vrfs[vrf].networks[network].route_map is arista.avd.defined %}
       network {{ network }} route-map {{ router_bgp.vrfs[vrf].networks[network].route_map }}
-{%                      else %}
+{%             else %}
       network {{ network }}
-{%                      endif %}
-{%                  endfor %}
-{%              endif %}
-{%             if router_bgp.vrfs[vrf].neighbors is defined and router_bgp.vrfs[vrf].neighbors is not none %}
-{%                 for neighbor in router_bgp.vrfs[vrf].neighbors %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].remote_as is defined and router_bgp.vrfs[vrf].neighbors[neighbor].remote_as is not none %}
+{%             endif %}
+{%         endfor %}
+{%         if router_bgp.vrfs[vrf].neighbors is arista.avd.defined %}
+{%             for neighbor in router_bgp.vrfs[vrf].neighbors %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].remote_as is arista.avd.defined %}
       neighbor {{ neighbor }} remote-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].remote_as }}
-{%                      elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is defined and router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is not none %}
+{%                 elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is arista.avd.defined %}
       neighbor {{ neighbor }} peer group {{ router_bgp.vrfs[vrf].neighbors[neighbor].peer_group }}
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined and router_bgp.vrfs[vrf].neighbors[neighbor].local_as is number %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined and router_bgp.vrfs[vrf].neighbors[neighbor].local_as is number %}
       neighbor {{ neighbor }} local-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].local_as }} no-prepend replace-as
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].description is defined and router_bgp.vrfs[vrf].neighbors[neighbor].description is not none %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].description is arista.avd.defined %}
       neighbor {{ neighbor }} description {{ router_bgp.vrfs[vrf].neighbors[neighbor].description }}
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is arista.avd.defined %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is arista.avd.defined %}
       neighbor {{ neighbor }} ebgp-multihop {{ router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is number else none }}
```

Comment:
No need for this extra inline jinja test. Just render whatever is in ebgp_multihop like we do everywhere else.

Reply:
updated to a cli generator as this is valid config: `neighbor 10.0.0.10 ebgp-multihop`